pkgs/tools/compression/ncompress/default.nix

```diff
@@ -1,15 +1,13 @@
 {lib, stdenv, fetchurl}:
 
 stdenv.mkDerivation rec {
-  name = "ncompress-4.2.4.6";
+  name = "ncompress-5.0";
```

Comment:
that could be converted to pname + version.

Reply:
Yes, it could. I will add a new PR (as this one is closed)